autoJoin.py

```diff
@@ -109,7 +130,16 @@ def check_and_end_or_leave_or_join_meeting():
             else:
                 browser.execute_script("document.getElementById('roster-button').click()")
         if curParticipants <= minParticipants and curParticipants != 0:  # leaves meeting for given condition
+
+                #twillion message Sending 
+            client.messages.create(
+            to='"'+data['your_no']+'"', 
+            from_='"'+data['twillio_no']+'"',
+            body='"'+"Hello "+data['nickname']+ " I left  the meeting named ** "+ meeting_name+"** as it wasn't ended becuase the current peoples in meeting were less than  **  "+ curParticipants + "** at the time of **"+format(datetime.now())+'**"')
```

Comment:
*because the current *people in meeting were less than  **  "+ ~~curParticipants~~ minParticipants